packages/web-api-scan-job-manager/src/worker/worker.spec.ts

```diff
@@ -39,6 +38,7 @@ describe(Worker, () => {
     let serviceConfigMock: IMock<ServiceConfiguration>;
     let loggerMock: IMock<MockableLogger>;
     let batchPoolLoadSnapshotProviderMock: IMock<BatchPoolLoadSnapshotProvider>;
+    let onDemandPageScanRunResultProviderMock: IMock<OnDemandPageScanRunResultProvider>;
```

Comment:
i don't see onDemandPageScanRunResultProviderMock.verifyAl() call anywhere. Without this call, the verifiable setups won't be validated.

Reply:
`afterEach(() => {
        batchMock.verifyAll();
        queueMock.verifyAll();
        serviceConfigMock.verifyAll();
        batchPoolLoadSnapshotProviderMock.verifyAll();
        systemMock.verifyAll();
        onDemandPageScanRunResultProviderMock.verifyAll();
    });`